README.md

```diff
@@ -37,4 +37,44 @@ The configuration file follows a simple structure.
         ...
     ]
 }
-```
\ No newline at end of file
+```
+
+### Adding variable type
+
+You can specify the type of a variable by extending its name with the type in the query as such: ```variableName_variableType```. The underscore ```_``` here is crutial to make a clear distinction between name and type. 
+
+### Representation Mapper 
+
+If you want to add your own type representations you can do this by adding your representation to the [typeMapper.js](./src/typeMapper.js) file. 
```

Comment:
Why would I want to add my own type representations? Can you give an example?

Reply:
I have added both in [commit 259d097](https://github.com/SolidLabResearch/generic-data-viewer/pull/6/commits/259d0979b5cee6cb7d7348032e3ab3404323d93a)